WordPress/Classes/ViewRelated/Blog/DiscussionSettingsViewController.swift

```diff
@@ -83,8 +83,8 @@ public class DiscussionSettingsViewController : UITableViewController
         }
         
         let service = BlogService(managedObjectContext: settings.managedObjectContext)
-        service.updateSettingsForBlog(settings.blog,
-            success: nil,
+        service.updateSettingsForBlog(blog,
+            success: { _ in },
```

Comment:
The [success closure is optional](https://github.com/wordpress-mobile/WordPress-iOS/blob/issue/blog-service-nullability/WordPress/Classes/Services/BlogService.m#L266). How about if we, instead, update the method signature, and avoid sending a NOP closure?